python_modules/libraries/dagster-celery-k8s/dagster_celery_k8s/executor.py

```diff
@@ -222,6 +239,7 @@ def _submit_task_k8s_job(app, plan_context, step, queue, priority, known_state):
         job_config_dict=job_config.to_dict(),
         job_namespace=plan_context.executor.job_namespace,
         user_defined_k8s_config_dict=user_defined_k8s_config.to_dict(),
+        per_step_k8s_config=per_step_k8s_config,
```

Comment:
`per_step_k8s_config=plan_context.executor.per_step_k8s_config` would allow you to do this without needing the partial